go/src/koding/db/models/groupdata.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// GroupData holds a group's private info.
+type GroupData struct {
+	ID   string `json:"_id"`
+	Slug string `json:"slug"`
+	Data *Data  `bson:"data,omitempty" json:"data,omitempty"`
```

Comment:
1. Less indirections, no need for nil-checks. This part from Get() can be removed

```go
+	if d == nil {
 +		return "", ErrDataKeyNotExists
 +	}
```

And all Get, GetString etc. methods could be defined on the value receiver (`func (d Data) Get(...`).

Also:

```
If the receiver is a map, func or chan, don't use a pointer to them. If the receiver is a slice and the method doesn't reslice or reallocate the slice, don't use a pointer to it.
```
https://github.com/golang/go/wiki/CodeReviewComments#receiver-type

2. 👍